src/SonarScanner.MSBuild.PreProcessor/ArgumentProcessor.cs

```diff
@@ -144,12 +144,9 @@ private static bool AreParsedArgumentsValid(ProcessedArgs args, ILogger logger)
                 areValid = false;
             }
 
-            if (!args.IsOrganizationValid)
-            {
-                areValid = false;
-            }
-
-            if (args.SonarServer is null)
+            if (!args.IsOrganizationValid
+                || args.SonarServer is null
+                || !args.IsJavaConfigurationValid)
```

Comment:
We also need to exit, if SkipJreProvisioning wasn't set properly.

Reply:
It's the same for both parameters.